src/pages/deploy/deployments.md

```diff
@@ -102,4 +110,10 @@ deployment in progress.
 
 If a deployment is completed, you can delete a live deploy by clicking the the
 three dots at the end of the deployment tab and select Remove. This will remove
-the deployment and stop any further project usage. 
+the deployment and stop any further project usage.
+
+### Deployment Restarts
```

Comment:
see my comment below - I think this section should probably just be "What happens if my deployment crashes?" or something like that

Reply:
Agree- made the requisite changes and added a photo.